api/v1/infinispan_types.go

```diff
@@ -175,6 +175,12 @@ type DiscoverySiteSpec struct {
 	// Enables (default) or disables the Gossip Router pod and cross-site services
 	// +optional
 	LaunchGossipRouter *bool `json:"launchGossipRouter,omitempty"`
+	// Memory resource request for Gossip Router (if LaunchGossipRouter == true)
```

Comment:
I would replace the "( if Launch.." with "...request for Gossip Router if enabled". Same applies for CPU.